src/transport.rs

```diff
@@ -230,6 +234,7 @@ pub struct ClientConnection {
 
     channels: Option<MessageChannels>,
     channels_rx: Option<IncomingMultiplexedPackets<MultiplexedPacket>>,
+    #[allow(dead_code)]
```

Comment:
Here we need comment too.

Reply:
ah yes, thanks. amended to add comment in both places